examples/address_generator.py

```diff
@@ -111,4 +111,19 @@ def output_seed(seed):
         'If not specified, the first unused address will be returned.'
   )
 
+  parser.add_argument(
+    '--security',
+      type      = int,
+      default   = 2,
+      help      = 'Security level to be used for the private key / address. Can be 1, 2 or 3',
+  )
```

Comment:
This is a great idea!  Good thinking!

Minor thing:  Could we replace the `2` with `AddressGenerator.DEFAULT_SECURITY_LEVEL`?